sensor/common/centralclient/testdata/update-testdata.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# This script will update the testdata for the TLSChallenge tests.
+# Usage: ./update-testdata.sh
+#
+# Prerequisites:
+# - roxcurl
+# - deployed StackRox instance via ./deploy/k8s/deploy-local.sh
+# - jq
+# - openssl
+
+if ! kubectl -n stackrox get secrets central-tls; then
+    echo "Central CA not found. Running StackRox instance with provisioned Central CA cert required."
+    exit 1
+fi
+
+# Usage: retry <max_retries> <delay_seconds> <function_to_retry>
+retry() {
+    local max_retries=$1
+    local delay=$2
+    local func=$3
+    local retry_count=0
+
+    while [ $retry_count -lt "$max_retries" ]; do
+        echo "Attempt $((retry_count + 1))"
+
+        # Execute and check the exit status of the function
+        if $func; then
+            echo "Function succeeded."
+            return 0
+        else
+            ((retry_count++))
+            echo "Function failed. Retrying in $delay seconds..."
+            sleep "$delay"
+        fi
+    done
+
+    echo "Max retries reached. Exiting."
+    return 1
+}
+
+trustInfoResponse=""
+exec_tls_challenge() {
+    trustInfoResponse=$(roxcurl "/v1/tls-challenge?challengeToken=h83_PGhSqS8OAvplb8asYMfPHy1JhVVMKcajYyKmrIU=")
```

Comment:
Please add a comment how this token was generated in case it needs to be regenerated at some point in the future.

Reply:
done